.gitattributes

```diff
@@ -1,5 +1,4 @@
 * text=auto
-*.cs text eol=crlf
```

Comment:
This ended up to be the issue with CI. The Analyzer doesn't like it on "comment lines".